elastic/apm-server
# apmpackage: bump version to 0.2.0

## Motivation/summary

Bump integration package version to 0.2.0

## How to test these changes

In Kibana navigate to Fleet, and find the "Elastic APM" integration. Check that its version is 0.2.0.

## Related issues

None.
